extensions/BLENDER_physics/schema/node.BLENDER_physics.schema.json

```diff
@@ -29,16 +29,73 @@
             "type": "boolean",
             "default": false
         },
-        "dimensions" : {
-            "description": "The bounding box dimensions of the node",
+        "bounding_box" : {
+            "description": "The local bounding box dimensions of the node",
             "type": "array",
             "items": {"type": "number"},
             "minItems": 3,
             "maxItems": 3
         },
-        "mesh": {
+        "radius" : {
+            "description": "Radius to use for calculating the physics shape based on bounding box of the node",
```

Comment:
The "based on bounding box of the node" wording seems a little confusing to me. I feel like it can be read as the radius is calculated from the bounding box or that the radius and bounding box are used to calculate a physics shape.